.releaserc.json

```diff
@@ -14,6 +14,12 @@
         "releaseRules": [{
           "type": "chore",
           "release": "patch"
+        },{
+          "type": "oaiFix",
```

Comment:
Does this mean commits with type oaiFix would go in `patch` realease and so on? 
Also, for commits, in general, should we follow a format? Something like <type>:<message> ?